index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav class="header">
+        <h3>Welcome</h3>
+        <a href=""><img src="images/PNG/Enabled.png" alt="Hamburger-icon"></a>
+    </nav>
+    <div class="title">
+        <h2 class="title1">Hey there. I'm Francisco<br><span class="title2">I'm a software developer</span></br></h2>
+        <p>I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don't hesitate to contact me.</p>
+        <nav class="socials">
+            <a href="#"><img src="images/PNG/twitter.png" alt="twitter-icon"></a>
+            <a href="#"><img src="images/PNG/Linkedin icon.png" alt="linkedin-icon"></a>
+            <a href="#"><img src="images/PNG/medium.png" alt="medium-icon"></a>
+            <a href="#"><img src="images/PNG/github.png" alt="github-icon"></a>
+            <a href="#"><img src="images/PNG/angellist.png" alt="angellist-icon"></a>
+        </nav>
+        <nav>
+            <a class="action" href="#"><img src="images/PNG/Action.png" alt="down-arrow"></a>
+        </nav>
```

Comment:
- Kindly replace the `div` tag for a `section` tag. It represents a generic standalone section of the document and should be wrapped inside a `section` tag. :nerd_face: :nerd_face:  
- Kindly replace the `h2` tag with `h1` tag. If you want that google (and other search engines) knows about a specific topic, in this case, you, use an `h1` tag, please. :+1: :smiley: :smiley: 
- The use of `a` tags for the icons is correct however please replace the `nav` tag with a `ul` tag to group the icons, and use `li` tags per icon. This way will be semantically correct. :eyes: :eyes: 
- Kindly remove the `nav` tag on line `27` it is not necessary to wrap this element inside a `nav` tag.